src/YesSql.Core/Sql/SchemaBuilder.cs

```diff
@@ -69,11 +51,11 @@ public ISchemaBuilder CreateMapIndexTable(Type indexType, Action<ICreateTableCom
                     ;
 
                 table(createTable);
-                Execute(_commandInterpreter.CreateSql(createTable));
+                await ExecuteAsync(_commandInterpreter.CreateSql(createTable));
```

Comment:
@sebastienros I did not want to change existing logic in the PR. Maybe in another PR. but why do we need to create 3 different connections here? `Execute`, `CreateForeignKey`, and `AlterTable` all create different connection to the database. I think these 3 should be executed within a transaction on the same connection. Either way, even if we want to change this, I rather do that in a follow up PR just so we can track it separately in case we run into issues.

Reply:
Though, all these methods use the local `Connection` property, unless I missed something it's then using the same connection object.